.gitignore

```diff
@@ -128,6 +128,7 @@ yarn-error.log*
 
 # local env files
 .env.local
+.env.sample
```

Comment:
`.env.sample` should not be in the `.gitignore` since we need it to show up for other contributors when they are building the project.